sharding/nodesCoordinator/common_test.go

```diff
@@ -151,3 +153,105 @@ func getExpandedEligibleList(num int) []uint32 {
 func newValidatorMock(pubKey []byte, chances uint32, index uint32) *validator {
 	return &validator{pubKey: pubKey, index: index, chances: chances}
 }
+
+func TestSerializableShardValidatorListToValidatorListShouldErrNilPubKey(t *testing.T) {
+	t.Parallel()
+
+	listOfSerializableValidators := []*SerializableValidator{
+		{
+			PubKey:  nil,
```

Comment:
i think it's better to provide non default values in examples, so here it should be:
`PubKey: []byte("pubKey1")` or something similar

Reply:
applicable in all other places

Reply:
in this place, I used it as nil to get the ErrNilPubKey
I modified where I used it as []byte{}